exercises/grains/README.md

```diff
@@ -33,7 +33,7 @@ For installation and learning resources, refer to the
 Run the tests by executing the test script.
 
 ```
-$ groovy ./HelloWorldSpec.groovy
+$ groovy ./GrainsSpec.groovy
```

Comment:
Nope, that's fine. As long as we get the readme template fixed, it will come out right next time we generate the README.